go-controller/pkg/node/node.go

```diff
@@ -312,6 +307,11 @@ func (n *OvnNode) Start(wg *sync.WaitGroup) error {
 		return err
 	}
 
+	// create libovsdbClient
+	if sbClient, err = libovsdb.NewSBClient(n.stopChan); err != nil {
```

Comment:
same as the other comment, you already created a client in cmd/ovnkube.go so just pass it into start

Reply:
+1 I guess for now anyways I will rip this commit out until we decide its safe to use libovsdb in the node process